test/api/todos.coffee

```diff
@@ -184,3 +251,34 @@ describe "Todos", ->
           expect(body.err).to.equal "Task not found."
           done()
 
+describe "habits", ->
+
+  it "Creates a completed a habit when using up url", (done) ->
+    upHabit = undefined
+    request.post(baseURL + "/user/tasks/withUp/up").send(
+        type: "habit"
+        text: "testTitle"
+        complete: true
+        description: "testDesc"
+    ).end (res) ->
+      expectCode res, 200
+      request.get(baseURL + "/user/tasks/withUp")
+      .send().end (res) ->
+        upHabit = res.body
+        expect(upHabit.completed).to.equal true
+        done()
+
+  it "Creates an uncompleted todo when using down url", (done) ->
```

Comment:
The complete/incomplete state was part of the issue request. So, I used test to mimic what the user was doing. They were confused between the creation of a habit vs todo when setting the complete status and using the up url.